CONTRIBUTING.md

```diff
@@ -2,42 +2,34 @@
 
 ## How to contribute
 
-If you would like to contribute to this project, see [Contributions to Splunk][indivcontrib] for more information.
+If you would like to contribute to this project, see [Contributions to Splunk](https://www.splunk.com/en_us/form/contributions.html) for more information.
 
-## Issues & Bug Reports
+## Issues and bug reports
 
-If you're seeing some unexpected behavior with this project, please create an [issue on GitHub][issues] with the following information:
+If you're seeing some unexpected behavior with this project, please create an [issue](https://github.com/splunk/splunk-sdk-python/issues) on GitHub with the following information:
 
 1. Version of this project you're using (ex: 1.5.0)
-1. Platform version (ex: Windows Server 2012 R2)
-1. Framework version (ex: Python 2.7.9)
-1. Splunk version (ex: 6.3.0)
-1. Other relevant information (ex: local/remote environment, Splunk network configuration)
+2. Platform version (ex: Windows Server 2012 R2)
+3. Framework version (ex: Python 2.7.9)
+4. Splunk version (ex: 6.3.0)
+5. Other relevant information (ex: local/remote environment, Splunk network configuration)
 
-Alternatively, if you have a Splunk question please ask on [Splunk Answers][answers]
+Alternatively, if you have a Splunk question please ask on [Splunk Answers](https://community.splunk.com/t5/Splunk-Development/ct-p/developer-tools).
 
 ## Pull requests
 
 We love to see pull requests!
 
 To create a pull request:
 
-1. Fill out the [Individual Contributor Agreement][indivcontrib].
-1. Fork [the repository][repo].
-1. Make changes to the **`develop`** branch, preferably with tests.
-1. Create a [pull request][pulls] against the **`develop`** branch.
+1. Fill out the [Individual Contributor Agreement](https://www.splunk.com/en_us/form/contributions.html).
+1. Fork the [repository](https://github.com/splunk/splunk-sdk-python).
+1. Make changes to the **develop** branch, preferably with tests.
+1. Create a [pull request](https://github.com/splunk/splunk-sdk-python/pulls) against the **develop** branch.
 
 ## Contact us
 
-You can [contact support][contact] if you have Splunk related questions.
+Contact [Support](https://www.splunk.com/en_us/support-and-services.html) if you have questions.
```

Comment:
"If you have a paid Splunk Enterprise or Splunk Cloud license, you can contact..."